client/my-sites/site-settings/main.jsx

```diff
@@ -17,31 +18,44 @@ import GeneralSettings from './section-general';
 
 import './style.scss';
 
+const getTitle = ( isClassicView ) => {
+	if ( isClassicView ) {
+		return translate( 'Settings' );
+	}
+	return translate( 'General Settings' );
+};
+
+const getSubtitle = ( isClassicView ) => {
+	if ( isClassicView ) {
+		return translate( 'Manage your site settings, including site visibility, and more.' );
```

Comment:
Copy could use work after removing the actual "site visibility" option from here fee4f0366849efd2554851c43b081103081f87d9